src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/CustomSelectionNodeModelNodeViewCustomization.cs

```diff
@@ -0,0 +1,35 @@
+using Dynamo.Controls;
+using Dynamo.Wpf;
+using System;
+using CoreNodeModels.Input;
+using CoreNodeModelsWpf.Controls;
+
+
+namespace CoreNodeModelsWpf.Nodes
+{
+    /// <summary>
+    /// View customizer for DropDown node model.
+    /// </summary>
+    public class CustomSelectionNodeModelNodeViewCustomization : INodeViewCustomization<CustomSelectionNodeModel>
+    {
+        public void CustomizeView(CustomSelectionNodeModel model, NodeView nodeView)
+        {
+            nodeView.inputGrid.Children.Add(
+                new CustomSelectionControl()
+                {
+                    DataContext = model
+                });
+        }
+
+        public void Dispose()
```

Comment:
this standard pattern for disposal does not really make sense when we're not actually cleaning up any native resources. Theres no reason to suppress the GC for a fully managed type. I'd just leave it empty for now.